extensions/spotify-beta/package.json

```diff
@@ -0,0 +1,111 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "spotify-beta",
+  "title": "Spotify Beta",
+  "description": "Spotify's most common features, now at your fingertips. Search for music and podcasts, browse your library, and control the playback. Glance at what's currently playing directly from the menu bar.",
+  "icon": "extension-icon.png",
+  "author": "mattisssa",
+  "contributors": [
+    "peduarte",
+    "sxn",
+    "dillionverma",
+    "andreaselia",
+    "stuart",
+    "tonka3000",
+    "dancannon"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "search",
+      "title": "Search",
+      "subtitle": "Spotify",
+      "description": "A single unified search command. Use this to search for artists, albums, songs, playlists, podcasts, and episodes. Use the dropdown menu to filter your search to a specific category. Each category offers contextual actions, so you can dive deeper into the search.",
+      "mode": "view"
+    },
+    {
+      "name": "yourLibrary",
+      "title": "Your Library",
+      "subtitle": "Spotify",
+      "description": "Use this to see your saved artists, albums, songs, playlists, and podcasts. Similar to the \"Search\" command, it includes a category dropdown and contextual actions.",
+      "mode": "view"
+    },
+    {
+      "name": "nowPlaying",
+      "title": "Now Playing",
+      "subtitle": "Spotify",
+      "description": "See what's currently playing. Use the actions (⌘ K) for further actions, such as Play/Pause, Like/Dislike, Skip, Start Radio, Add to Playlist, Connect Device and more.",
+      "mode": "view",
+      "preferences": [
```

Comment:
Why not make it a global preference? We could use this for actions in `Your Library` or `Search`.

Reply:
Mainly because I was imagining that I'd only have these type of actions (like, dislike, play, pause, add to playlist) in Now Playing